axelrod/tests/unit/test_player.py

```diff
@@ -20,7 +20,9 @@ class TestPlayerClass(unittest.TestCase):
 
     name = "Player"
```

Comment:
Aaaah... Yes. I suppose we could create another specific method for this class, that would then also be inherited but is never called? I'm just thinking that it would be possible for someone to change something in init that breaks things...